### PR TITLE
Extend to support any scrollable container

### PR DESCRIPTION
**Pull request by koshov**

### CODE REVIEW

File: lib/ReduxInfiniteScroll.js [hunk 1]
```diff
@@ -67,7 +69,7 @@ var ReduxInfiniteScroll = function (_React$Component) {
   }, {
     key: '_elScrollListener',
     value: function _elScrollListener() {
-      var el = _reactDom2.default.findDOMNode(this);
+      var el = this.props.scrollableElement || document.getElementById('workspace');
```
Comment (Antho2407) [inline lib/ReduxInfiniteScroll.js]: do you think we can get rid of this workspace thing and open a PR for the library ? Would be nice to drop some open source contributions 😄

**Review by Antho2407 — APPROVED** [review]
Nice find ! Would be really nice to drop a PR for the lib though

**Review by MomchilGorchev — APPROVED** [review]
No too much context, but the code looks good

File: lib/ReduxInfiniteScroll.js (hunk 1 above)
Comment (koshov) [reply lib/ReduxInfiniteScroll.js]: Good catch - this is not meant to be there at all! 
Yeah I'll prepare the PR, however the author hasn't responded in 6 months and this is based on 2 pending PRs for the library already :\

**Pull request merged**